examples/kubernetes/connectivity-check/README.md

```diff
@@ -37,3 +37,28 @@ multiple files per the following logic:
   directory.
 
 For more information, see https://github.com/cilium/cilium/pull/12599 .
+
+## Listing and generating connectivity checks
+
+```
+$ make help
+List connectivity-check resources specified in this directory
+
+Usage:
+  cue [-t component=<component>] [-t kind=<kind>] [-t name=<name>] [-t quarantine=true] [-t topology=<topology>] [-t traffic=any] [-t type=<tooltype>] <command>
```

Comment:
this is really helpful 👍 .

Just a quick question on this. `make help` output suggests to run `cue` command directly, meanwhile our Makefile is running cue in docker. 

So do you feel that we need to add one small step to install cue binary in local system (e.g. similar to what we have for golangci-lint, ginkgo, gomega, etc) ?